src/component/geo/GeoView.js

```diff
@@ -39,9 +39,10 @@ export default echarts.extendComponentView({
             return;
         }
 
-        var mapDraw = this._mapDraw;
         if (geoModel.get('show')) {
-            mapDraw.draw(geoModel, ecModel, api, this, payload);
+            this._mapDraw = new MapDraw(api, true);
+            this.group.add(this._mapDraw.group);
+            this._mapDraw.draw(geoModel, ecModel, api, this, payload);
```

Comment:
Don't create MapDraw each render. It may cause performance issue since some diff optimization can't be applied.
Also diff transition can't be applied.